src/utils/safe-messages.ts

```diff
@@ -38,7 +54,6 @@ export const generateSafeMessageHash = (safe: SafeInfo, message: SafeMessage['me
 
 export const supportsEIP1271 = ({ version, fallbackHandler }: SafeInfo): boolean => {
   const EIP1271_SUPPORTED_SAFE_VERSION = '1.0.0'
-  // From v1.3.0, EIP-1271 support was moved to the CompatibilityFallbackHandler
   const EIP1271_FALLBACK_HANDLER_SUPPORTED_SAFE_VERSION = '1.3.0'
```

Comment:
`EIP1271_FALLBACK_HANDLER_SUPPORTED_SAFE_VERSION` is now declared twice. The other constant `EIP1271_SUPPORTED_SAFE_VERSION` could be moved to the top of the file.